launchdarkly-server-sdk/Cargo.toml

```diff
@@ -50,6 +51,7 @@ reqwest = { version = "0.12.4", features = ["json"] }
 [features]
 default = ["rustls"]
 rustls = ["hyper-rustls/http1", "hyper-rustls/http2", "eventsource-client/rustls"]
+compress = ["flate2"]
```

Comment:
It might make sense to call this `event-compression` or something, just so it's clear that this is relevant only to events and not flag payloads (which seems like something people might also be looking for.)

Reply:
I did rename this as requested.